public/app/features/dashboard/containers/PublicDashboardPageProxy.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { config } from '@grafana/runtime';
+import { GrafanaRouteComponentProps } from 'app/core/navigation/types';
+
+import { PublicDashboardScenePage } from '../../dashboard-scene/pages/PublicDashboardScenePage';
+
+import PublicDashboardPage from './PublicDashboardPage';
+import { PublicDashboardPageRouteParams, PublicDashboardPageRouteSearchParams } from './types';
+
+export type PublicDashboardPageProxyProps = GrafanaRouteComponentProps<
+  PublicDashboardPageRouteParams,
+  PublicDashboardPageRouteSearchParams
+>;
+
+function PublicDashboardPageProxy(props: PublicDashboardPageProxyProps) {
+  if (config.featureToggles.dashboardSceneForViewers || props.queryParams.scenes) {
```

Comment:
I'd set this based on a new FF only. It will be easier to release and it can be enabled earlier than the "all dashboard running on scenes" FF.